README.md

```diff
@@ -68,6 +68,14 @@ Orca Slicer's logo is designed by community member Justin Levine(@freejstnalxndr
   - Tools needed: Xcode, Cmake, git, gettext, libtool, automake, autoconf, texinfo
       - You can install most of them by running `brew install cmake gettext libtool automake autoconf texinfo`
   - run `build_release_macos.sh`
+  - To build and debug in XCode:
+      - run `XCode.app`
+      - open `build_arm64/OrcaSlicer.xcodeproj`
```

Comment:
Maybe `build_<arch>` just for us folks still using intel macs 😺

Reply:
Done!